src/source-pfring.c

```diff
@@ -353,6 +399,14 @@ TmEcode ReceivePfringLoop(ThreadVars *tv, void *data, void *slot)
              * reset it here */
             PACKET_PROFILING_RESTART(p);
 
+#ifdef HAVE_PF_RING_FLOW_OFFLOAD
+            if (ptv->flags & PFRING_FLAGS_BYPASS) {
+                p->pfring_v.flow_id = hdr.extended_hdr.pkt_hash; /* pkt hash contains the flow id in this configuration */
```

Comment:
could you set Packet::flow_hash directly here?

Reply:
I'm not sure it's a good idea, just wondering. It does look like Suricata wouldn't use it yet in the flow table though, and I see it's being overwritten.

So please disregard :)